src/acceptance/broker/broker_test.go

```diff
@@ -164,80 +167,84 @@ var _ = Describe("AutoScaler Service Broker", func() {
 		})
 	})
 
-	It("should update service instance from autoscaler-free-plan to acceptance-standard", func() {
-		plans := getPlans()
-		if plans.length() < 2 {
-			Skip(fmt.Sprintf("2 plans needed, only one plan available plans:%+v", plans))
-			return
-		}
-		service := createService(plans[0])
-		service.updatePlan(plans[1])
-
-		By("delete service")
-		service.delete()
-	})
-
-	It("should fail to update service instance from acceptance-standard to first", func() {
-		plans := getPlans()
-		if plans.length() < 2 {
-			Skip(fmt.Sprintf("2 plans needed, only one plan available plans:%+v", plans))
-			return
-		}
-		if !plans.contains("acceptance-standard") {
-			Skip(fmt.Sprintf("Acceptance test standard plan required plans:%+v", plans))
-			return
-		}
-
-		service := createService("acceptance-standard")
-		updateService := service.updatePlanRaw(plans[0])
-		Expect(updateService).To(Exit(1), "failed updating service")
+	Describe("allows updating service plans", func() {
+		var instance serviceInstance
+		It("should update a service instance from one plan to another plan", func() {
+			servicePlans := GetServicePlans(cfg)
+			source, target, err := servicePlans.getSourceAndTargetForPlanUpdate()
+			if err != nil {
+				Skip(err.Error())
```

Comment:
Why are we skipping this test in case we don't have the broker properly setup with test plans that allow updating? 

Since we can configure the broker deployed during a CI run, we don't we simply rely on it that we've configured it properly?

Reply:
We know that customers might run the acceptance tests on their deployments and have previously complained about tests cases that cannot run with their specific configuration.

I have now skipped them via `test_default`: 170e5999c2e919ecfdebc1926e93d7218f0e0759